src/main/java/com/hivemq/extensions/rbac/configuration/entities/ExtensionConfig.java

```diff
@@ -36,14 +35,12 @@ public class ExtensionConfig {
     @XmlElement(name = "credentials-reload-interval", defaultValue = "60")
     private int reloadInterval = 60;
 
-    @Nullable
     @XmlElementWrapper(name = "listener-names")
     @XmlElement(name = "listener-name")
-    private Set<String> listenerNames;
+    private @Nullable Set<String> listenerNames;
 
-    @NotNull
     @XmlElement(name = "password-type", defaultValue = "HASHED")
-    private PasswordType passwordType = PasswordType.HASHED;
+    private @Nullable PasswordType passwordType = PasswordType.HASHED;
```

Comment:
how can this get null?

Reply:
It is possible to generate cases with JAXB where writing a wrong value will result in null.